### DISCUSSION TITLE
Filter entries table by one or more related categories

### DISCUSSION THREAD

**Discussion by angrybrad** [craftcms/cms#9483] [Ideas] [answered]
> Created by: Daniel Howells (daniel.howells@gmail.com) on 2015/11/23 13:00:18 +0000
> Votes at time of UserVoice import: 2

-----------------------------------

It would be great if entries could be filtered by one or more related categories where applicable, in addition to the standard keyword search

**Comment by pixelmachine** [comment]
This was added to the 3.6/4.0 milestone but now it's moved to discussion is that still the case?

**Reply by brandonkelly** [reply]
Yep, on the list for 4.0.

**Comment by brandonkelly** [comment, marked as answer]
This has been implemented for Craft 4 🎉 (#10075)
